src/panels/config/devices/device-detail/ha-device-entities-card.ts

```diff
@@ -125,6 +141,51 @@ export class HaDeviceEntitiesCard extends LitElement {
     fireEvent(this, "hass-more-info", { entityId: entry.entity_id });
   }
 
+  private async _addToLovelaceView(): Promise<void> {
+    if ((this.hass!.panels.lovelace?.config as any)?.mode === "yaml") {
+      alert(
```

Comment:
Isn't this a bit lame. Can't we just disable the button in that case ? 

Or, open a dialog with a textarea that includes the yaml ?

Reply:
Yeah, I'm planning to add that functionality